source/_components/sensor.bme680.markdown

```diff
@@ -0,0 +1,155 @@
+---
+layout: page
+title: "BME680 Sensor"
+description: "Instructions how to integrate a BME680 sensor into Home Assistant."
+date: 2018-01-16 00:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: raspberry-pi.png
+ha_category: Sensor
+ha_release: 0.62
+ha_iot_class: "Local Push"
+---
+
+
+The `bme680` sensor platform allows you to read temperature, humidity, pressure and gas resistance values of a [Bosch BME680 Environmental sensor](https://cdn-shop.adafruit.com/product-files/3660/BME680.pdf) connected via [I2c](https://en.wikipedia.org/wiki/I²C) bus (SDA, SCL pins). It allows you to use all the operation modes of the sensor described in its datasheet.  In addition, it includes a basic air quality calculation that uses gas resistance and humidity measurements to calculate a percentage based air quality measurement.
+
+Tested devices:
+
+- [Raspberry Pi](https://www.raspberrypi.org/)
+
+To use your BME680 sensor in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: bme680
+```
+
+Configuration variables:
```

Comment:
Please use the configuration tags, for more information please see:
https://home-assistant.io/developers/documentation/create_page/#configuration